src/components/Main/Main.js

```diff
@@ -19,13 +19,13 @@ const Main = ({ isLoading }) => {
     fetchedMovieContent
   ) : (
     <main className="main">
-      <div className="main__separator" />
-      <div className="main__container">
-        <div className="main__container__tools">
+      <div className="separator" />
+      <div className="main-container">
+        <div className="main-container__tools">
```

Comment:
The longer the classNames, the more difficult it gets to look at the template, and since this doesn't really matter if this is a part of `main` or a `main-container`, let's use the shorter version.
```suggestion
        <div className="main__tools">
```